soc/renesas/rcar/rcar_gen3/Kconfig.defconfig

```diff
@@ -5,11 +5,9 @@
 
 if SOC_SERIES_RCAR_GEN3
 
-config SOC_SERIES
-	default "gen3"
-
 config NUM_IRQS
-	default 512
+	default 512 if SOC_R8A77951_R7
+	default 240 if SOC_R8A77961 || SOC_R8A77951_A57
```

Comment:
you may align the number of IRQs with R7, actually it is the same according to documentation 480 SPI and 32 PPI+SGI